src/lib/search-and-replace.js

```diff
@@ -150,7 +150,10 @@ export const searchAndReplace = async (
 	}
 
 	// determine all the replacements required
-	const replacementsArr = pairs.map( str => str.split( ',' ) );
+	const replacementsArr = pairs.map( str => {
+		const filtered = str.replace( /[\s\x00-\x1F\x7F]/g, '' ); //whitespace & char codes 0-32, 127.
+		return filtered.split( ',' );
```

Comment:
Is this better than using `trim()` directly on the strings? Something like:

```js
replacementsArr.map( str => str.split( ',' ).map( s => s.trim() ) );
```

```

Reply:
Good suggestions. Two reasons I didn't go forward for that:
1) It won't remove spaces or special chars from _within_ the string
2) trim only removes whitespace and not the other non-printables. 

But it feels like I'm overthinking this, so I'll probably refactor a bit along the lines of your suggestion.

Reply:
oh, thank you for working on that! Looks good to me